exampleSite/config.toml

```diff
@@ -28,6 +28,7 @@ defaultContentLanguageInSubdir = false
   office_hours = "Monday 10:00 to 13:00 or email for appointment"
   phone = "888 888 88 88"
   skype = "echo123"
+  telegram = "LenaSmith"
```

Comment:
Change to `telegram = ""` so that Telegram linking is disabled by default (otherwise, if we later add other chat links, there will be too many links)